pkg/apis/serving/v1alpha1/knativeserving_types.go

```diff
@@ -38,6 +45,8 @@ type KnativeServingSpec struct {
 	// A means to override the corresponding entries in the upstream configmaps
 	// +optional
 	Config map[string]map[string]string `json:"config,omitempty"`
+
+	Image Registry `json:"registry,omitempty"`
```

Comment:
I think we should include the proper updates to `deploy/crds/serving_v1alpha1_knativeserving_crd.yaml` with this PR. Either manually or via `operator-sdk generate k8s`

Reply:
Done